pyuvdata/uvdata/uvh5.py

```diff
@@ -767,6 +1074,7 @@ def write_uvh5(
         check_extra=True,
         run_check_acceptability=True,
         clobber=False,
+        chunks=True,
```

Comment:
This is a nice addition. To make sure it's accessible to the user, it should be added to `UVData.write_uvh5` as well.